JPF/PubAdvWebScrape.ipynb

```diff
@@ -0,0 +1,605 @@
+       "589               2709 HEATH AVENUE BRONX        RICH  LAUBSCH  346    7\n",
+       "590         100 LINDEN BOULEVARD BROOKLYN          JASON  KORN  117    1\n",
+       "591        1616 PRESIDENT STREET BROOKLYN          JASON  KORN   98    0\n",
+       "592                1311 AVENUE K BROOKLYN          JASON  KORN  134    0\n",
+       "593            31 EAST 1 STREET MANHATTAN        AMANDA  REYES   84    0\n",
+       "594          328 WEST 47 STREET MANHATTAN        AMANDA  REYES   87    8\n",
+       "595          330 WEST 47 STREET MANHATTAN        AMANDA  REYES   39    0\n",
+       "596          412 WEST 49 STREET MANHATTAN        AMANDA  REYES  128    0\n",
+       "597          68 WEST 126 STREET MANHATTAN         MENDEL  GOLD  139    4\n",
+       "598           1183 FULTON STREET BROOKLYN         MENDEL  GOLD   51   14\n",
+       "599          73 JEFFERSON STREET BROOKLYN         MENDEL  GOLD   31    0\n",
+       "600            729 MEEKER AVENUE BROOKLYN         MENDEL  GOLD   26    9\n",
+       "601          327 NOSTRAND AVENUE BROOKLYN         MENDEL  GOLD   13    0\n",
+       "602          1316 PACIFIC STREET BROOKLYN         MENDEL  GOLD   30    8\n",
+       "603         274 VAN BUREN STREET BROOKLYN         MENDEL  GOLD   10    9\n",
+       "604           820 EAST 10 STREET BROOKLYN      EPHRAIM  LANDAU  147    2\n",
+       "605           2626 KINGS HIGHWAY BROOKLYN      EPHRAIM  LANDAU  188    2\n",
+       "606         1181 EASTERN PARKWAY BROOKLYN        JOSEPH  EMILE   94    0\n",
+       "607         1187 EASTERN PARKWAY BROOKLYN        JOSEPH  EMILE   95    2\n",
+       "608         318 ROCHESTER AVENUE BROOKLYN        JOSEPH  EMILE  147    0\n",
+       "609         539 WEST 148 STREET MANHATTAN         DAVID  BOWEN   81    0\n",
+       "610             814 MACON STREET BROOKLYN         DAVID  BOWEN   29    2\n",
+       "611          758 NOSTRAND AVENUE BROOKLYN         DAVID  BOWEN   43    0\n",
+       "612          820 NOSTRAND AVENUE BROOKLYN         DAVID  BOWEN   20    0\n",
+       "613            654 PUTNAM AVENUE BROOKLYN         DAVID  BOWEN   50    0\n",
+       "614      319 MALCOLM X BOULEVARD BROOKLYN         DAVID  BOWEN   26    1\n",
+       "615  419 MARCUS GARVEY BOULEVARD BROOKLYN         DAVID  BOWEN   35    0\n",
+       "616  421 MARCUS GARVEY BOULEVARD BROOKLYN         DAVID  BOWEN   49    0\n",
+       "\n",
+       "[606 rows x 4 columns]"
+      ]
+     },
+     "execution_count": 70,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from selenium import webdriver\n",
+    "from bs4 import BeautifulSoup\n",
+    "import requests\n",
+    "import bs4\n",
+    "import pandas as pd\n",
+    "from bs4 import BeautifulSoup\n",
+    "from selenium.webdriver.common.desired_capabilities import DesiredCapabilities\n",
+    "from selenium import webdriver\n",
```

Comment:
Selenium is super heavy for simple web scraping. Any reason we did just use something simple, like [requests](http://docs.python-requests.org/en/master/)?